src/components/SelectButtonGroup/SelectButton/SelectButton.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+interface Props {
+    /** the label to be displayed on this button */
+    children: string;
+    /** the value associated with this button */
+    value: string;
+    /**
+     * A function to be called if a button is pressed.
+     * This prop, if defined on the parent SelectButtonGroup will overwrite this one
+     */
+    onSelect?: (value: string) => void;
+    /** if this button should be in a selected state */
+    isSelected?: boolean;
+    /** the color context to be applied in the selected state */
+    selectedContext?: 'neutral' | 'brand';
+    /**
+     * is this button part of a multiselect group
+     * when yes will render as checkbox otherwise as radioButton
+     * This prop will be overwritten by the parent <SelectButtonGroup>
+     */
+    isMultiselect?: boolean;
```

Comment:
I think we don't need `isMultiselect` for SelectButton component. It already exists in the parent. It can cause problems. For example, if set for one `isMultiselect=true`, and for others `isMultiselect=false` so what will be output?

Reply:
We need it because that what determines if it will render as a radio button or as a checkbox. It is not really meant to be specified by the user of the API but passed down from the parent, as says in the comments above. However, we do need a way to pass it down, so we need this prop to exist on the button itself.

Reply:
Can't it perhaps be passed on through `...rest`? Just like we do in other components, without having validation? This way they're not documented in Storybook as "public" props.

Reply:
It is not about how it is passed, it is about how it is received.

Reply:
I understand that, but how come we need to have these props be explicitly defined (while they are not meant to be applied on this level by the user) while in other components we use `rest` to apply any other props without having such explicit definitions or validation?

I.e., can you explain why we cannot apply those props that are passed down from the parent using `rest`, without defining them explicitly on child level (again, since they're not part of the actual component API)?

Reply:
I think we don't need to use `checkbox`/`radio`. It can be:
```
<select  multiple>
  <option selected>Open this select menu</option>
  <option value="1">One</option>
  <option value="2">Two</option>
  <option value="3">Three</option>
</select>

```
this cover two scenarios and we can rid of `isMultiselect` for item

Reply:
That's a very good idea!